### PR TITLE
@ #4 | should add k8s-stack-cookbook stuff with chef json config

### PR DESCRIPTION
**Pull request by hieptranquoc**
connect #4

### CODE REVIEW

**Comment by hieptranquoc** [comment]
@hoatle review this , plz

**Comment by hieptranquoc** [comment]
@hoatle i can't assign this PR for you ...

**Review by hoatle — CHANGES_REQUESTED** [review]
this is funny, do you get the configuration work yet?

Just copy and past my requirement is not acceptable, I need the implementation guy.

**Comment by hieptranquoc** [comment]
@hoatle fixed! please review again

File: chef/main-cookbooks/teracy/Berksfile [hunk 1]
```diff
@@ -2,3 +2,5 @@
 source 'https://supermarket.chef.io'
 
 metadata
+
+cookbook "kubernetes-stack", git: "https://github.com/teracyhq-incubator/kubernetes-stack-cookbook.git", branch: "develop"
```
Comment (hoatle) [inline chef/main-cookbooks/teracy/Berksfile]: use the latest release from Chef Supermarket instead

File: vagrant_config_default.json [hunk 2]
```diff
@@ -19,12 +19,31 @@
...
     "_ua_cookbooks_path": [
+      "workspace/dev-setup/chef/berks-cookbooks",
       "workspace/dev-setup/chef/main-cookbooks"
     ],
     "_ua_run_list": [
       "teracy"
     ],
     "json": {
+      // to work with kubernetes
+      "kubernetes-stack": {
```
Comment (hoatle) [inline vagrant_config_default.json:30]: have you checked to make sure everything should work? I haven't seen the reciple (k8s.rb) to handle the  json config here yet. (check the similar setup on acme101/dev-setup

**Review by hoatle — CHANGES_REQUESTED** [review]

File: chef/main-cookbooks/teracy/Berksfile (hunk 1 above)
Comment (hieptranquoc) [reply chef/main-cookbooks/teracy/Berksfile]: @hoatle cookbook 'kubernetes-stack', '~> 0.1.1' !

Comment (hieptranquoc) [reply chef/main-cookbooks/teracy/Berksfile]: fixed!

File: vagrant_config_default.json (hunk 2 above)
Comment (hieptranquoc) [reply vagrant_config_default.json:30]: fixed!

**Comment by hieptranquoc** [comment]
@hoatle fixed! Check again please :D

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**